.golangci.yml

```diff
@@ -4,7 +4,6 @@ run:
 linters:
   disable-all: true
   enable:
-    - deadcode
```

Comment:
obviously we can just yeet these, but they do still _run_. 
Are there new linters that replace their function? (so we don't lose the coverage)

Reply:
It looks like the "unused" check is doing that, which is an actual name of a linter we use, so I don't think it's saying the check has no replacement.

Reply:
Hrm double negatives are difficult

Reply:
To hopefully phrase it better, in the output from the linter:

```
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
```

I believe `unused` is the name of the replacement, and we already have that enabled.

Reply:
_oh_